201-vmss-existing-vnet/azuredeploy.json

```diff
@@ -133,16 +142,16 @@
                       "name": "ipconfig",
                       "properties": {
                         "subnet": {
-                          "id": "[resourceId(parameters('existingVnetResourceGroupName'), 'Microsoft.Network/virtualNetworks/subnets', parameters('existingVnetName'), parameters('existingSubNetName'))]"
+                          "id": "[resourceId(parameters('existingVnetResourceGroupName'), '/Microsoft.Network/virtualNetworks/subnets/', parameters('existingVnetName'), parameters('existingSubNetName'))]"
```

Comment:
I think all of these are left over from using concat() instead of resourceId, so need a bit of TLC before deployment will succeed.

Reply:
Sorry but I don't understand what should I do...